modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -33,47 +31,43 @@ $ oc apply -f cw-secret.yaml
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
-  serviceAccountName: <service_account_name> <3>
+  serviceAccountName: 
+    name: <service_account_name> <1>
   outputs:
-   - name: cw <4>
-     type: cloudwatch <5>
+   - name: cw <2>
```

Comment:
in v6 we require 3 or more chars for a name.   Probably best to find a new example name.

Reply:
updated